src/prefect/environments/execution/dask/k8s.py

```diff
@@ -18,12 +18,10 @@
 from prefect.utilities import logging
 
 
-class CloudEnvironment(Environment):
+class DaskKubernetesEnvironment(Environment):
     """
-    CloudEnvironment is an environment which deploys your flow (stored in a Docker image)
-    on Kubernetes and it uses the Prefect dask executor by dynamically spawning workers as pods.
-
-    *Note*: This environment is not currently customizable. This may be subject to change.
+    DaskKubernetesEnvironment is an environment which deploys your flow (stored in a Docker image)
```

Comment:
Could you add something here mentioning that this is backed by (and based on) the functionality of https://kubernetes.dask.org/en/latest/